### PR TITLE
in docs navigation, make partitions independent from schedules

### PR DESCRIPTION
**Pull request by sryza**
### Summary & Motivation

The fact that we bucket partitions with schedules makes users think that we discourage use of non-time-window partitions.

### CODE REVIEW

File: docs/content/_navigation.json [hunk 1]
```diff
@@ -144,7 +144,8 @@
         ]
       },
       {
-        "title": "Schedules, Sensors, & Partitions",
+        "title": "Schedules & Sensors",
+        "path": "/concepts/partitions-schedules-sensors/schedules",
```
Comment (tacastillo) [inline docs/content/_navigation.json:148]: We should probably also split the directories in either your upstream PR or this PR, probs the other one to minimize conflicts.

**Review by tacastillo — APPROVED** [review]
I ❤️ this change.

I'd probably wait for Erin before merging since she owns information architecture.

File: docs/content/_navigation.json (hunk 1 above)
Comment (erinkcochran87) [reply docs/content/_navigation.json:148]: I agree we need to do this eventually, but I recommend leaving it alone for now. Moving directories is a pretty involved process at the moment. The IA of the docs is likely to change fairly drastically in the next few months as we release the tutorial and new guides. I'm don't know that many people would clock this if we leave it, but we can re-assess if it does cause issues.

**Review by erinkcochran87 — APPROVED** [review]

File: docs/content/_navigation.json (hunk 1 above)
Comment (sryza) [reply docs/content/_navigation.json:148]: 👍

**Pull request merged**